### ISSUE TITLE
Pre-Train the Judge Model and determine which one is best

### ISSUE THREAD

**Issue by MrinalA2009** [alexliao95311/DebateSim#127]
Give the Judge agent a "rubric" for judging, and then make a supervised learning feedback loop to improve its capabilities of judging and feedback till we are satisfied. Potential loss function: Log Loss (binary classification to decide aff or neg).
